src/mlpack/methods/ann/layer/c_relu.hpp

```diff
@@ -0,0 +1,151 @@
+/**
+ * @file c_relu_impl.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of CReLU layer.
+ * Introduced by,
+ * Wenling Shang, Kihyuk Sohn, Diogo Almeida, Honglak Lee,
+ * "https://arxiv.org/abs/1603.05201", 16th March 2016.
```

Comment:
Can you use a similar comment/cite style as used here:

https://github.com/mlpack/mlpack/blob/60fb72eb377901337bca7708e57a038d83226c23/src/mlpack/methods/ann/layer/dropconnect.hpp#L27-L51

Reply:
Made apt changes